cg/meta/orders/microbial_fastq_submitter.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+
+from cg.constants import Workflow, DataDelivery
+from cg.exc import OrderError
+from cg.meta.orders.lims import process_lims
+from cg.meta.orders.submitter import Submitter
+from cg.models.orders.order import OrderIn
+from cg.models.orders.sample_base import StatusEnum
+from cg.store.exc import EntryNotFoundError
+from cg.store.models import Sample, Customer, Case, CaseSample
+from cg.utils.error_handler import handle_errors
+
+
+class MicrobialFastqSubmitter(Submitter):
+    @handle_errors(to_except=(EntryNotFoundError,), to_raise=OrderError)
+    def submit_order(self, order: OrderIn) -> dict:
+        project_data, lims_map = process_lims(
+            lims_api=self.lims, lims_order=order, new_samples=order.samples
+        )
+        status_data: dict = self.order_to_status(order)
+        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
+        new_samples: list[Sample] = self.store_items_in_status(
+            customer_id=status_data["customer"],
+            order=status_data["order"],
+            ordered=project_data["date"] if project_data else datetime.now(),
+            ticket_id=order.ticket,
+            items=status_data["samples"],
+        )
+        self._add_missing_reads(new_samples)
+        return {"project": project_data, "records": new_samples}
+
+    @staticmethod
+    def order_to_status(order: OrderIn) -> dict:
+        """Convert order input for microbial samples."""
+        return {
+            "customer": order.customer,
+            "order": order.name,
+            "samples": [
+                {
+                    "application": sample.application,
+                    "comment": sample.comment,
+                    "data_analysis": sample.data_analysis,
+                    "data_delivery": sample.data_delivery,
+                    "name": sample.name,
+                    "priority": sample.priority,
+                    "volume": sample.volume,
+                    "control": sample.control,
+                }
+                for sample in order.samples
+            ],
+        }
+
+    def store_items_in_status(
+        self,
+        customer_id: str,
+        order: str,
+        ordered: datetime,
+        ticket_id: str,
+        items: list[dict],
+    ) -> list[Sample]:
+        customer: Customer = self._get_customer(customer_id)
+        new_samples: list[Sample] = []
+        for sample in items:
+            case_name: str = sample["name"]
```

Comment:
Will this get confusing? Should we add a prefix or something to have it different than the sample name?